doc/ceph-volume/lvm/activate.rst

```diff
@@ -19,10 +19,88 @@ instead.
 New OSDs
 --------
 To activate newly prepared OSDs both the :term:`OSD id` and :term:`OSD uuid`
-need to be supplied. For example::
+need to be supplied,or only term:`OSD uuid` need to be supplied.  
+For example use `OSD id` and  `OSD uuid`::
```

Comment:
Suggest:
To activate newly prepared OSDs with ``ceph-volume``, one must provide the :term:`OSD uuid` and optionally the :term:`OSD id`.